src/node.cpp

```diff
@@ -142,4 +148,29 @@ std::vector<session_key_t> Node::getReadySessions() const {
     return std::move(readySessions);
 }
 
+Status Node::demultiplyOutputs(SessionResults& nodeSessionOutputs) {
+    auto& [metadata, blobMap] = nodeSessionOutputs.begin()->second;
+    auto& [session, blob] = *blobMap.begin();
+    std::vector<NodeSessionMetadata> newSessionMetadatas(metadata.generateSubsessions(getName(), demultiplexCount.value()));
+    auto tensorDesc = blob->getTensorDesc();
+    auto newDims = tensorDesc.getDims();
+    newDims.erase(newDims.begin() + 1);
+    const InferenceEngine::TensorDesc dividedBlobDesc(
+        tensorDesc.getPrecision(),
+        newDims,
+        InferenceEngine::Layout::ANY);
+    const auto step = blob->byteSize() / demultiplexCount.value();
+    for (size_t i = 0; i < newSessionMetadatas.size(); ++i) {
+        InferenceEngine::Blob::Ptr dividedBlob;
+        auto status = createSharedBlob(dividedBlob, dividedBlobDesc);
+        if (!status.ok()) {
+            return status;
+        }
+        memcpy((char*)dividedBlob->buffer(), (char*)blob->buffer() + i * step, step);
```

Comment:
We should check if dividedBlob buffer size is equal to `step` before copying memory